broadcast/skins/broadcast/src/coffee/roca.coffee

```diff
@@ -95,7 +113,17 @@
       target.data 'roca-url', url
       target.loading().reload()
       if (el.data 'roca-trap-submit') is 'yes'
-        target.funnelSubmit()
+        options = { "onCompleteEvent": el.data 'roca-submit-complete-event' }
+        target.funnelSubmit(options)
+
+      # Optionally reload container when specific events are fired, delayed by
+      # a specified time
+      event = el.data 'roca-refresh-on'
+      if event
+        delay = el.data 'roca-refresh-delay'
+        refreshOn target, event, delay
```

Comment:
This is all that is needed for the auto refresh. What event is triggered and how, is not this plugin's concern. This plugin should just worry about what even to listen for.